### ISSUE TITLE
Question: Can't write to mounted devices (acl)

### ISSUE THREAD

**Issue by fogine** [coldfix/udiskie#143]
Hello there,

I'm not able to write data to mounted devices even after I tweak file ACLs for current user (not root). Note that I don't have write permission even with `sudo` elevated user.
I realize this is not probably `udiskie` issue at all, however I struggle with resolving the issue.  
Could I ask for your help?
[HERE](https://unix.stackexchange.com/questions/379148/udisks2-acl-cant-get-write-permissions-for-a-regular-user) is corresponding stackexchange question, that I created, with more details...  

**EDIT**:
Interesting is that on my work computer (with exact same setup and same permissions & acl settings) I'm able to write to mounted devices... And If I remember correctly I was able to write to the devices about a week ago on my personal computer.  
Do you have an idea what could went wrong when using `udiskie` ?

I have been using automount with  `udiskie-umount --detach /run/media/myuser/device` command.

Thank you for your useful contributions to OS!

**Comment by coldfix** [comment]
Hi,

sorry for the late response, somehow I forgot to answer.

can you check whether the device is mounted as read-only (type `mount` to see flags). If the device is mounted as readonly, check your `~/.config/udiskie/config.yml` whether your config sets any devices to be mounted as readonly. If the config is not to blame, please run in verbose mode (e.g. `udiskie-mount -v /dev/sdb1`) to see what's going on when the device is mounted.

Also, please provide the output of `udisksctl info -b /dev/sdb1` (using the proper block device).

If readonly is not the problem:

Are you sure you properly checked that you can't write in sudo mode?

Note that this is **NOT** a proper check because the redirection is performed as user:

```bash
sudo echo foo > /run/media/myuser/device/file
```

Use this instead:

```bash
sudo touch /run/media/myuser/device/file
```

If this was the problem, try to `chown myuser:myuser` or `chmod 777` on the mount path.

**Comment by fogine** [comment]
Thanks for your response @coldfix ,

while debugging I noticed that only `ntfs` formated devices could not be written to.
After reading [THIS](https://unix.stackexchange.com/questions/132273/why-cant-i-write-on-external-hard-disk) stackexchange question and installing the `ntfs-3g` package on my arch system, udiskie mounts ntfs devices with 777 permissions and I can actually write to the device.  
I still don't get where I should have noticed this / known about it...
@coldfix , can it be determined from the debug information I gathered by following udiskie [debugging](https://github.com/coldfix/udiskie/wiki/Debugging-a-problem) guide before I installed the `ntfs-3g` package? As everything I've done (before I installed the `ntfs-3g` package) was telling me I should have had write permissions to the device.
Data available [HERE](https://gist.github.com/fogine/ae74d14dc52dd2d9e85bc2e7eee8a232)

This issue can be considered solved.
